deno.jsonc

```diff
@@ -0,0 +1,42 @@
+{
+  "unstable": ["sloppy-imports"],
+  "workspace": [
+    "packages/ajv-validator",
+    "packages/arktype-validator",
+    // "packages/auth-js",
```

Comment:
What's the reason you comment out the packages?

Reply:
They fail `deno publish --dry-run` for various reasons. I had thought we could merge the PR and revisit them later?

Reply:
@BarryThePenguin Ahh, understood. Okay!

So, can I merge this PR right now?

Reply:
Let's go!